dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -33,7 +32,7 @@ import { formatDateTime } from "utils/dateFunctions";
 const SavedRunsComponent = () => {
   const dispatch = useDispatch();
   const { savedRuns, selectedRuns } = useSelector((state) => state.overview);
-  const loginDetails = useSelector((state) => state.userAuth.loginDetails);
+  // const loginDetails = useSelector((state) => state.userAuth.loginDetails);
```

Comment:
Can we just remove this code instead of leaving it commented out?

Reply:
Commented out for future use.

Reply:
See my previous comment.

Reply:
Removed it.